.gitlab-ci.yml

```diff
@@ -233,26 +232,20 @@ integration:builds:
   script:
     - mkdir -p ./builds
     - >
-      nix-shell --arg ci true --run $'
-      npmDepsHash="$(prefetch-npm-deps ./package-lock.json)";
-      build_application="$(nix-build \
+      nix develop .#ci --command bash -c $'
+      build_application="$(nix build \
         --max-jobs "$(nproc)" --cores "$(nproc)" \
-        ./release.nix  \
-        --attr application \
-        --argstr npmDepsHash "$npmDepsHash" \
+        .#polykey-cli \
       )";
       nix-store --export $( \
         nix-store --query --requisites "$build_application" \
       ) | gzip > ./builds/polykey-cli.closure.gz;
-      builds="$(nix-build \
+      builds="$(nix build \
         --max-jobs "$(nproc)" --cores "$(nproc)" \
-        ./release.nix \
-        --attr docker \
-        --attr package.linux.x64.elf \
-        --attr package.windows.x64.exe \
-        --attr package.macos.x64.macho \
-        --argstr npmDepsHash "$npmDepsHash" \
-        --argstr commitHash "$CI_COMMIT_SHA")";
+        .#docker \
+        .#packages.x86_64-linux \
+        .#packages.x86_64-windows \
+        .#packages.x86_64-darwin;
```

Comment:
I want to redo the naming here... `docker`, `linux.x64.elf`. Why change the names here?

Reply:
The names are changed here, because when the flake builds for each of those platforms, it will output the relevant binary with the appropriate file extension for the platform.

Reply:
We don't want the file extension, none of the binaries should have a file extension.

Reply:
The binaries do not have a file extension as of commit a9ba3b2 and a7a1351.